README.md

```diff
@@ -1,36 +1,50 @@
-# docker-invenio
+# CentOS 7 with Python 3 for Invenio
 
-[![Build Status](https://travis-ci.org/inveniosoftware/docker-invenio.svg?branch=master"BuildStatus\")](https://travis-ci.org/inveniosoftware/docker-invenio/branches?branch=master) [![image](https://img.shields.io/docker/automated/inveniosoftware/centos7-python.svg)](https://hub.docker.com/r/inveniosoftware/centos7-python/) [![image](https://img.shields.io/docker/build/inveniosoftware/centos7-python.svg)](https://hub.docker.com/r/inveniosoftware/centos7-python/builds/)
+[![Build Status](https://travis-ci.org/inveniosoftware/docker-invenio.svg?branch=master)](https://travis-ci.org/inveniosoftware/docker-invenio) [![image](https://img.shields.io/docker/automated/inveniosoftware/centos7-python.svg)](https://hub.docker.com/r/inveniosoftware/centos7-python/) [![image](https://img.shields.io/docker/build/inveniosoftware/centos7-python.svg)](https://hub.docker.com/r/inveniosoftware/centos7-python/builds/)
 
-This repository contains Docker base images to generate the environment for
-[Invenio](https://github.com/inveniosoftware/invenio) applications.
+This image serves as base image for [Invenio](https://github.com/inveniosoftware/invenio) instances running on CentOS7. The purpose is to provide a base image that is usable in production environments like OpenShift.
 
-**Prerequisites** Docker: [https://docs.docker.com/install/](https://docs.docker.com/install/)
+The image is based on the official CentOS image ``centos:7`` and contains:
+
+- Python 3.6 set as default Python interpreter with upgraded versions of pip, pipenv, setuptools and wheel.
+- Tools: Node.js, NPM, Git, Curl Vim, Emacs, Development Tools.
+- Library devel packages: libffi, libxml2, libxslt.
+- Working directory for an Invenio instance.
+
+## Supported tags and respective ``Dockerfile`` links
+
+* 3.6 - [Dockerfile](https://github.com/inveniosoftware/docker-invenio/blob/master/python3.6/Dockerfile).
```

Comment:
Maybe we should warn people that we are not pushing `latest` tag, they are going to get the following if they do not specify the `3.6` one:
```Dockerfile
FROM inveniosoftware/centos7-python
RUN echo 1
```
```console
$ docker build . -t the-invenio
Sending build context to Docker daemon  687.1kB
Step 1/2 : FROM inveniosoftware/centos7-python
manifest for inveniosoftware/centos7-python:latest not found
```
Or:
```console
docker run -ti --rm inveniosoftware/centos7-python /bin/bash
Unable to find image 'inveniosoftware/centos7-python:latest' locally
docker: Error response from daemon: manifest for inveniosoftware/centos7-python:latest not found.
See 'docker run --help'.
```

Reply:
At line 56 it's written:  "This way, we will use tag `3.6` as latest for Python version and there is also the possibility to push tags for specific use cases such as pinning certain libraries or patches."

We can specify here that we don't push the `latest` tag.